src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpCertificateHelper.cs

```diff
@@ -55,11 +55,11 @@ internal static class WinHttpCertificateHelper
             unsafe
             {
                 var cppStruct = new Interop.Crypt32.CERT_CHAIN_POLICY_PARA();
-                cppStruct.cbSize = (uint)Marshal.SizeOf<Interop.Crypt32.CERT_CHAIN_POLICY_PARA>();
+                cppStruct.cbSize = (uint)sizeof(Interop.Crypt32.CERT_CHAIN_POLICY_PARA);
```

Comment:
What are the rules regarding when it is safe to use `sizeof()` vs. `Marshal.SizeOf<T>()` ?

Reply:
Basically you can use sizeof unless there is marshaling logic involved that would change the size of the type, e.g. if a field was attributed with MarshalAs that would convert it from one thing to another of a different size during the marshaling process.  If the type is blittable, sizeof is the way to go.

Reply:
It is safe to use `sizeof` for [blittable types](https://en.wikipedia.org/wiki/Blittable_types). Both `sizeof()` and `Marshal.SizeOf<T>()` return same value for blittable types, except that sizeof is faster.

We have some of the best practices for interop perf documented in: https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md#definition